src/wallet/rpcwallet.cpp

```diff
@@ -2716,7 +2716,7 @@ UniValue mint(const UniValue& params, bool fHelp)
 
     if (fHelp || params.size() != 1)
         throw std::runtime_error(
-            "mint amount\n"
+            "mint <amount>(could not smaller than 0.1)\n"
```

Comment:
I think we should use the same format as Bitcoin RPC. So it should be something like `mint amount\n`

Reply:
make sense.